ott/core/quad_problems.py

```diff
@@ -493,6 +498,17 @@ def update_epsilon_unbalanced(epsilon, transport_mass):
   return updated_epsilon
 
 
+def update_geom_scale_cost(geom: Optional[geometry.Geometry],
```

Comment:
maybe this should be in geometry.py and take self as first argument?

Reply:
Originally, I was against this (as the method either returns unmodified `self` or a new instance), but I changed my mind and think your suggestion is better.
Done as a private method `_set_scale_cost(self, scale_cost)`.